src/renderer/store/updateTranscriptionAPIKey/reducer.ts

```diff
@@ -0,0 +1,20 @@
+import { Reducer } from 'redux';
+import { Action } from '../action';
+import { ApplicationStore, initialStore } from '../sharedHelpers';
+import { UPDATE_TRANSCRIPTION_API_KEY_TOGGLED } from './actions';
+
+const updateTranscriptionAPIKeyReducer: Reducer<
+  ApplicationStore['updateTranscriptionAPIKeyOpened'],
+  Action<boolean>
+> = (
+  updateTranscriptionAPIKeyOpened = initialStore.updateTranscriptionAPIKeyOpened,
+  action
+) => {
+  if (action.type === UPDATE_TRANSCRIPTION_API_KEY_TOGGLED) {
+    return action.payload as boolean;
```

Comment:
More of a styling thing, but I've seen a lot of casting done as

`<boolean>action.payload`

elsewhere in the project. I don't think it's worth a change but worth mentioning, as maybe we can standardise it or something idk

Reply:
`as` is the more common one in our project IMO, this is fine